src/main/resources/application-prod.properties

```diff
@@ -1,7 +1,9 @@
-#-------------------Overwriting prod mongo config---------------
+#----------------- Overwriting prod mongo config---
 spring.data.mongodb.database=${DB_NAME}
 spring.data.mongodb.host=${DB_CONTAINER_NAME}
 spring.data.mongodb.username=${DB_USERNAME}
 spring.data.mongodb.password=${DB_PASSWORD}
-#----------------- Custom ----------------------
-filefighter.disable-password-check=false
\ No newline at end of file
+#----------------- Custom --------------------------
+filefighter.disable-password-check=false
+#----------------- ReverseProxyOnly ----------------
+server.servlet.context-path=/api/
```

Comment:
where is this being used?

Reply:
that is the base url. -> on e.g `/` an error appears, but that cannot be reached from proxy